ui/shared/ShortcutTable.jsx

```diff
@@ -54,6 +54,15 @@ const ShortcutTable = function ShortcutTable() {
               </td>
               <td>Open the logviewer for the selected job</td>
             </tr>
+            <tr>
+              <td>
+                <kbd>q</kbd>
+              </td>
+              <td>
+                Toggle showing only classified, unclassified, retried and
+                usercancelled jobs
```

Comment:
I'd reword this as: `Toggle between unclassified failures and all failures (including retried and user cancelled jobs)`